.gitignore

```diff
@@ -8,3 +8,6 @@ wwwroot/
 node_modules/
 *.user
 .pnpm-debug.log
+
+# Mac
+.DS_Store
```

Comment:
Since this file is common among our projects, also update it in the template with a similar pull request: https://github.com/Lombiq/Open-source-template